tests/unit/amo/api/test_index.js

```diff
@@ -631,6 +631,7 @@ describe(__filename, () => {
 
       await _fetchAddon({ slug: 'foo' }).then(unexpectedSuccess, (error) => {
         expect(error.message).toMatch(
+          // eslint-disable-next-line prefer-regex-literals
```

Comment:
Because the `RegExp` isn't actually being used as a `RegExp` (i.e., it is just a string with no special `RegExp` characters), we can just pass the string to `toMatch`. I.e., we can remove the `new RegExp` and then we won't need to disable the rule either. This is likely the case for other instances of this below.